src/server/rpc/libs/nasa.xml

```diff
@@ -0,0 +1 @@
+<blocks app="NetsBlox 0.7.1, http://netsblox.org" version="1"><block-definition s="Astronomy Pic of the Day Data" type="reporter" category="variables"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NASA</l><l>apod</l><l></l></block></block></script></block-definition><block-definition s="Latest Mars Weather Data" type="reporter" category="variables"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NASA</l><l>latestMarsWeather</l><l></l></block></block></script></block-definition></blocks>
```

Comment:
I think you forgot to include the `APOD pic` rpc block that you use in the example